static/package-lock.json

```diff
@@ -8596,7 +8596,10 @@
       "version": "1.3.1",
       "resolved": "https://registry.npmjs.org/full-icu/-/full-icu-1.3.1.tgz",
       "integrity": "sha512-VMtK//85QJomhk3cXOCksNwOYaw1KWnYTS37GYGgyf7A3ajdBoPGhaJuJWAH2S2kq8GZeXkdKn+3Mfmgy11cVw==",
-      "dev": true
```

Comment:
was this file meant to be added?

Reply:
npm keeps adding this deps. This should be safe and only used in dev mode.